src/scripts.js

```diff
@@ -51,12 +51,38 @@ const hydrationDate = document.querySelector('#hydrationDate');
 const hydrationResponse = document.querySelector('#hydrationResponse');
 const hydrationForm = document.querySelector('#hydrationForm');
 
+const stepSwivelButton = document.querySelector('#stepSwivelButton');
+const sleepSwivelButton = document.querySelector('#sleepSwivelButton');
+const hydrationSwivelButton = document.querySelector('#hydrationSwivelButton');
+const stepCardInner = document.querySelector('#stepCardInner');
+const sleepCardInner = document.querySelector('#sleepCardInner');
+const hydrationCardInner = document.querySelector('#hydrationCardInner');
+const stepUnswivelButton = document.querySelector('#stepUnswivelButton');
+const sleepUnswivelButton = document.querySelector('#sleepUnswivelButton');
+const hydrationUnswivelButton = document.querySelector('#hydrationUnswivelButton');
+
 // event listeners
 window.addEventListener('load', displayData);
 sleepButton.addEventListener('click', checkForSleepInputs);
 hydrationButton.addEventListener('click', checkForHydrationInputs);
+stepSwivelButton.addEventListener('click', toggleSwivel);
```

Comment:
Instead of having all these lines of code I believe we can do something like so:

swivelButton.forEach(button => {
  button.addEventListener('click', toggleSwivel)
});

Reply:
Also we might have to tweak the function a bit.

Reply:
Awesome catch! It felt so clunky, but I didn't even consider your suggestions. They work perfectly. Everything is updated and has been pushed up.

Thanks for paying close attention Ricardo and for calling out gross code!!